commands/price.mjs

```diff
@@ -77,6 +77,8 @@ const defaultFunction = {
                 }
             }
 
+            let tierPrice = item.avg24hPrice;
```

Comment:
@Razzmatazzz 

This line needed to be `item.avg24hPrice;` instead of `item.avg24Price;` - Super easy fix 😄